core/backend/src/GeoCoordConfig.ts

```diff
@@ -0,0 +1,89 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module iModels
+ */
+
+// cspell:ignore customuri
+
+import { BentleyError, Logger } from "@itwin/core-bentley";
+import { IModelHost } from "./IModelHost";
+import { Settings } from "./workspace/Settings";
+import { WorkspaceContainer, WorkspaceDb } from "./workspace/Workspace";
+
+const loggerCat = "GeoCoord";
+
+interface GcsDbProps extends WorkspaceDb.Props, WorkspaceContainer.Alias {
+  prefetch?: boolean;
+  priority?: number;
+}
+
+export class GeoCoordConfig {
```

Comment:
missing release tag on this API.

Reply:
@kabentley api-extractor presumably didn't catch this because this file's contents are not exported from core-backend.ts. Please suggest an appropriate release tag.

Reply:
this is all internal and purposely un-exported from this package. It should not need a release tag.